C++/segment_tree.cpp

```diff
@@ -1,224 +0,0 @@
-// Program to show segment tree to demonstrate lazy 
```

Comment:
@rahulranjan74  This file should not be deleted.

Reply:
> @rahulranjan74 This file should not be deleted.

yeah sure